modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/ChainExecutor.java

```diff
@@ -72,7 +75,7 @@ private void handleSuccess(CoreEvent childEvent, Consumer<Result> successHandler
 
   private CoreEvent handleError(Throwable error, CoreEvent childEvent, BiConsumer<Throwable, Result> errorHandler) {
     try {
-      errorHandler.accept(error, EventedResult.from(childEvent));
+      errorHandler.accept(error, org.mule.runtime.module.extension.api.runtime.privileged.EventedResult.from(childEvent));
```

Comment:
For some reason INtelliJ was refusing otherwise. In either case, probably for some ambiguity. Also, since the entire bug was because of picking the wrong import, I figured it didn't hurt

Reply:
As you wish, however the import seems to be unused now.